src/selectors/getChildren.ts

```diff
@@ -74,3 +74,31 @@ export const getChildrenRanked = (state: State, context: Context): Child[] =>
 /** Returns the first visible child of a context. */
 export const firstVisibleChild = (state: State, context: Context) =>
   getChildrenSorted(state, context)[0]
+
+/**
+ * Returns visible children.
+ *
+ * @param state - Redux state.
+ * @param path - Parent's path.
+ * @param simplePath - Parent's simple path.
+ * @param children - Child or ThoughtContext array.
+ * @param showContexts - Parent's context view status.
+ */
+export const filterChildren = (state: State, path: Path, simplePath: SimplePath, children: (Child | ThoughtContext)[], showContexts?: boolean) => {
```

Comment:
The comment and the name of the function need some work. `filterChildren` is too general: what kind of filter? Also see below comment about the appropriate function boundary for predicates.